cms/envs/common.py

```diff
@@ -921,6 +921,7 @@
     # other apps that are.  Django 1.8 wants to have imported models supported
     # by installed apps.
     'lms.djangoapps.verify_student',
+    'lms.djangoapps.grades',
```

Comment:
@jcdyer @sanfordstudent This line resolves https://github.com/edx/edx-platform/pull/13275